docs/running/service.rst

```diff
@@ -212,3 +212,43 @@ instead of adding any config to Mopidy add this to
 :file:`~mopidy/.pulse/client.conf`::
 
     default-server=127.0.0.1
+
```

Comment:
Add an extra newline here, just to follow the convention from the rest of the docs.

Reply:
Fixed in 36906f8f.